README.md

```diff
@@ -14,10 +14,12 @@ Provides OpenTelemetry Exporters for Google Cloud Operations.
 This project requires a mock server for Google Cloud APIs.  To build and test, do the following:
 
 ```
-$ source get_mock_server.sh
-$ ./gradlew test -Dmock.server.path=$MOCKSERVER
+$ ./gradlew test
 ```
 
+Note: This project uses [Test Containers](http://testcontainers.org), which requires
+docker to be runnable locally by the current users.  Please verify `docker busybox echo hello` works, and if not configure your local docker before building.
```

Comment:
```suggestion
docker to be runnable locally by the current users.  Please verify `docker run hello-world` works, and if not configure your local docker before building.
```
Either this, or `docker run -it --rm busybox echo hello` if you prefer terse output.

Reply:
I would +1 adding `--rm`